### ISSUE TITLE
Ability to load fonts from file

### ISSUE THREAD

**Issue by brendonparker** [QuestPDF/library#19]
**Is your feature request related to a problem? Please describe.**
I'm trying to deploy my code to an AWS lambda function. As such, I have limited control of the runtime, such as fonts. Apparently there are no fonts available. During render of the PDF, any text simply appears as blank.

**Describe the solution you'd like**
A way for me to be able to specify the font other than relying on `SKTypeface.FromFamilyName`

**Describe alternatives you've considered**
I'd have to work around this by adding a layer to the lambda, which will add some overhead and possibly impact cold start times.

**Additional context**
Originally, I was getting an error: 
```
Unable to load shared library 'libSkiaSharp' or one of its dependencies.
```

I fixed that by adding a dependency on `SkiaSharp.NativeAssets.Linux.NoDependencies`. That ensures `libSkiaSharp.so` gets published.

Expected output:
![image](https://user-images.githubusercontent.com/989122/131191241-a041c464-d84c-4c78-9c8d-2799f3a9cae4.png)

Actual output:
![image](https://user-images.githubusercontent.com/989122/131191258-23f7e22b-2fc4-4600-82a2-d0483ade3bbd.png)

**Comment by MarcinZiabek** [comment]
This is indeed a problem, thank you for all: creating this issue, investigation and the pull request. This essentially opens two topics: 

1) The ability to use custom fonts loaded from a stream and/or file (covered by your pull request).
2) The ability of a font fallback. So the library uses another font if the specified one is not available (or throws an exception to let the developer know).

Currently, I am working on a huge text rendering update. This is really complicated, far more than I expected so will take at least one more month to complete. I would like to publish a new minor version with a fix to 1) in the nearest week.

What do you think about the 2) case. Currently, I think the library should detect the missing font and just throw an exception. The font fallback may be great for web pages where we want to display at least "something". But in PDF generation, I don't want to create confusion about why the text on the document appears differently than the configured font. Any thoughts?

**Comment by brendonparker** [comment]
1. Great!
2. I think that is the intent of `SKFontManager.Default.MatchFamily` to provide a fallback if it can’t be found. I agree throwing an exception is more desirable than a blank render.

**Comment by MarcinZiabek** [comment]
Great news :) I have just published the new version of QuestPDF 2021.9: https://www.nuget.org/packages/QuestPDF/ 
Can you please download it and check if everything works for you?

**Comment by MarcinZiabek** [comment]
This feature was confirmed to work properly in another pull request. Closing.
